package.json

```diff
@@ -16,7 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "heroku-cli-util": "^8.0.8",
-    "heroku-exec-util": "0.7.4",
+    "heroku-exec-util": "https://github.com/heroku/heroku-exec-util#start-checking-host-keys",
```

Comment:
will this be ok in master?

Reply:
I just realised I messed up the PR description which may have confused things - I copy/pasted it from a similar PR but neglected to untick the checklist! Both of the items in the description should have been unticked - I've done that now. Apologies.

So to explain - this change is not intended for master - it's just in a temporary WIP commit so we can see the tests for this project passing with that new version. Once https://github.com/heroku/heroku-exec-util/pull/7 is merged and released I'll update the commit to the new release version (`0.7.5` probably), and _that's_ what'll go into master.